general/style-guides/index.md

```diff
@@ -0,0 +1,10 @@
+---
+uid: style-guides
+title: Style Guides
+---
+
+# Style Guides
+
+This section documents the code style used for the different languages used by Jellyfin.
```

Comment:
Might be worth noting that we provide [.editorconfig] files to enforce _some_ of the guidelines (like whitespaces & line endings).

[.editorconfig]: https://editorconfig.org/

Reply:
I do intend to mention that we have EditorConfig and ESLint files to help people stick to whatever will be in there :)